package.json

```diff
@@ -116,6 +116,7 @@
     "build:netlify": "CI=false npm run build && npm run sourcemaps:upload",
     "release": "npm version $npm_config_isomer_update && git push --tags",
     "test": "craco test",
+    "test:playwright": "source .env && npx playwright test --ui",
```

Comment:
Could we include the `reset-e2e` script when running playwright as well?

Reply:
sgtm